test/spec/features/popup-menu/PopupMenuSpec.js

```diff
@@ -251,69 +242,34 @@ describe('features/popup', function() {
     it('should attach popup to html', inject(function(popupMenu) {
 
       // when
-      popupMenu.open({}, 'menu' ,{ x: 100, y: 100 });
-
-      var container = popupMenu._current.container;
-
-      // then
-      expect(domClasses(container).has('djs-popup')).to.be.true;
-      expect(domClasses(container).has('menu')).to.be.true;
-    }));
-
-
-    it('should be visible even if no `position.cursor` was passed', inject(function(popupMenu) {
-
-      // when
-      popupMenu.open({}, 'menu' ,{ x: 100, y: 100 });
+      popupMenu.open({}, 'menu', { x: 100, y: 100 });
 
       var container = popupMenu._current.container;
+      var popup = container.children[0];
 
       // then
-      expect(getComputedStyle(container).visibility).not.to.eql('hidden');
+      expect(domClasses(container).has('djs-popup-parent')).to.be.true;
```

Comment:
We could make these tests more robust to verify that `domQueryAll('djs-popup', container)` returns a single element.